scripts/monotone-framework-lattice.py

```diff
@@ -0,0 +1,230 @@
+import argparse
+import sys
+from collections import defaultdict
+from pygraphviz import AGraph
```

Comment:
`pygraphviz` sucks. We're trying to get rid of it. Can you take a look if you can find some other (possibly pure-Python) library able to parse GraphViz stuff (without going full `graph_tool`)?

Reply:
refarctored to use `nextworkx` instead of `pygraphviz`. It uses `pydot` under the hood, which is pure-Python when used for plain dot parsing and not drawing